live-examples/js-examples/statement-async.html

```diff
@@ -0,0 +1,20 @@
+<pre>
+<code id="static-js">function resolveAfter2Seconds(x) {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(x);
+    }, 2000);
+  });
+}
+
+async function add1() {
+  console.log('calling');
+  var a = await resolveAfter2Seconds('resolved');
+
+  console.log(a);
+  // expected output: "resolved"
+}
+
+add1();
+</code>
```

Comment:
I think given that we're overlong already we should omit the blank line at 13. Also `add1` is now a weird name, and something like `result` might be better than `a` for the return value.

Also I wondered if the example would be a bit less weird if instead of taking `"resolved"` as an argument, it takes no arguments and just returns `"resolved"`:

```
function resolveAfter2Seconds() {
  return new Promise(resolve => {
    setTimeout(() => {
      resolve('resolved');
    }, 2000);
  });
}

async function asyncCall() {
  console.log('calling');
  var result = await resolveAfter2Seconds();
  console.log(result);
  // expected output: "resolved"
}

asyncCall();
```

Reply:
Makes sense. That's done now :)